diff --git a/index.ts b/index.ts
index b715298aa6b8f6ed02be24909850803d699e6711..3fa0a5419ae23bb58ef770906d9877a4eeab07bf 100644
--- a/index.ts
+++ b/index.ts
@@ -33,14 +33,6 @@ export default function starlightOpenAPIPlugin(userConfig: StarlightOpenAPIUserC
           customCss: [...(starlightConfig.customCss ?? []), 'starlight-openapi/styles'],
         }
 
-        if (updatedConfig.expressiveCode !== false) {
-          updatedConfig.expressiveCode =
-            updatedConfig.expressiveCode === true || updatedConfig.expressiveCode === undefined
-              ? {}
-              : updatedConfig.expressiveCode
-          updatedConfig.expressiveCode.removeUnusedThemes = false
-        }
-
         updateConfig(updatedConfig)
       },
     },
